src/Button/index.js

```diff
@@ -0,0 +1,117 @@
+// @flow strict
+
+import * as React from 'react';
+import { View } from 'react-native';
+import { defaultTokens } from '@kiwicom/orbit-design-tokens';
+
+import { StyleSheet, type StylePropType } from '..';
+import GenericButtonWrapper from './GenericButtonWrapper';
+import ButtonTitle from './ButtonTitle';
+import type { ButtonType } from './ButtonTypes';
+
+import Icon from '../native/Icon';
```

Comment:
This should be imported like
`import Icon from '@kiwicom/react-native-orbit-icons';`